content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,95 +1,82 @@
-=========================
-Create Bills of Materials
-=========================
+==========================
+Create a Bill of Materials
+==========================
 
-A *Bill of Materials* is a document defining the quantity of each
-component required to make or deliver a finished product. Additionally, it 
-can also include various operations and individual step guidelines
-needed to complete a production process. 
+A *Bill of Materials* (BoM) is a document that defines the quantity of each component required to
+make or deliver a finished product. It can also include various operations and individual step
+guidelines needed to complete a production process.
 
-With Odoo's MRP solution, multiple BOMs can also be linked to each product,
-so that even product variants can have their *own* tailored BOM.
+In Odoo Manufacturing, multiple BoMs can be linked to each product, so that even product variants
+can have their own tailored BoMs.
 
-This will help you to optimize your manufacturing process and save time 
-as a result. 
+Correctly setting up BoMs helps optimize the manufacturing process and save time.
 
 Setting up a BoM
 ================
-The simplest BoM set up is one without operations or instructions. 
-In this case, you will manage your production using Manufacturing Orders
-only.
+The simplest BoM setup is one without operations or instructions, only components. In this case,
+the production is solely managed using Manufacturing Orders. 
 
-To create a BoM from the Manufacturing module, go to :menuselection:`Products --> Bills of Materials`. 
-If you click **Create**, the first thing you'll have to do is to specify the final product. 
-For an existing product, simply select it from the list, or create a new one on the spot. Note 
-that if you go through the product form to create your BOM, the product will already be set for you.
+To create a BoM from the Manufacturing module, go to :menuselection:`Products --> Bills of
+Materials`. Then, click :guilabel:`Create`. Next, specify the :guilabel:`Product`. Note that a BoM
+can also be created directly from the product form. In that case, the :guilabel:`Product` field is
+pre-filled. 
 
-For a standard Bill of Material, keep the default *BoM* type, which is *Manufacture this Product*. 
-Now specify the various components that make up the production of your final product and their 
-respective quantities. You can create components as you go, on the BOM, or create products beforehand,
-by going to the :menuselection:`Top Menu --> Products --> Create`, and add them later on. 
+For a standard BoM, set the :guilabel:`BoM Type` to :guilabel:`Manufacture this Product`. Then
+click :guilabel:`Add a Line` to specify the various components that make up the production of the
+final product and their respective quantities. New components can be created on the fly through the
+BoM, or can be created beforehand in :menuselection:`Manufacturing --> Products --> Products
+--> Create`.
```

Comment:
Add `` Finally, click :guilabel:`Save` to finish creating the BoM. `` at the end of this paragraph